File: src/Day3/part1.go
package Day3

import (
  "strconv"
  "strings"
)

func Day3Part1(input string) (string, error) {
  lines := strings.Split(strings.TrimSpace(input), "\n")
  count := countTrues(lines, 3, 1)
  return strconv.Itoa(count), nil
}

File: src/Day3/part1.rs
use crate::day3::util::{count_trues};

pub fn day3part1(input: &String) -> Option<String> {
    let slope = input.lines().map(|x| x.to_string()).collect();
    let count = count_trues(&slope, 3, 1);
    return Some(count.to_string());
}
